src/options.ts

```diff
@@ -183,6 +183,19 @@ export async function loadOptions (configOverrides: NitroConfig = {}): Promise<N
     })
   }
 
+  // Normalize route rules
+  for (const rule of Object.values(options.routes)) {
+    if (rule.cors) {
+      rule.headers = {
+        ...rule.headers,
+        'access-control-allow-origin': '*',
+        'access-control-allowed-methods': '*',
+        'access-control-allow-headers': '*',
+        'access-control-max-age': '0'
+      }
```

Comment:
```suggestion
      rule.headers = {
        'access-control-allow-origin': '*',
        'access-control-allowed-methods': '*',
        'access-control-allow-headers': '*',
        'access-control-max-age': '0',
        ...rule.headers
      }
```

Reply:
I think allowing headers to override cors defaults is likely better, so users can edit the defaults